po/extra/pt.po

```diff
@@ -0,0 +1,50 @@
+# Portuguese (Portugal) translations for com.github.danrabbit.lookbook package.
+# Copyright (C) 2019 THE com.github.danrabbit.lookbook package.'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the extra package.
+# André Barata <andretiagob@protonmail.com>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: extra\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-05-14 18:27-0700\n"
+"PO-Revision-Date: 2020-08-27 19:16+0100\n"
+"Language-Team: none\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.4.1\n"
+"Last-Translator: André Barata <andretiagob@protonmail.com>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: pt\n"
+
+#: data/com.github.danrabbit.lookbook.appdata.xml.in:7
+#: data/com.github.danrabbit.lookbook.desktop.in:3
+msgid "LookBook"
+msgstr "LookBook"
+
+#: data/com.github.danrabbit.lookbook.appdata.xml.in:8
+#: data/com.github.danrabbit.lookbook.desktop.in:4
+msgid "Browse and find system icons"
+msgstr "Pesquise e encontre ícones de sistema"
+
+#: data/com.github.danrabbit.lookbook.appdata.xml.in:10
+msgid ""
+"Browse system icons by category or search by icon name. See icons at "
+"different sizes and get a quick code snippet to use in your apps."
+msgstr ""
+"Pesquise ícones de sistema por categoria ou procure por nome de "
+"ícone. Veja ícones em diferentes tamanhos e obtenha uma amostra de "
+"código para usar nas suas aplicações."
+
+#: data/com.github.danrabbit.lookbook.appdata.xml.in:21
+msgid "Daniel Foré"
+msgstr "Daniel Foré"
+
+#: data/com.github.danrabbit.lookbook.desktop.in:6
+msgid "com.github.danrabbit.lookbook"
+msgstr "com.github.danrabbit.lookbook"
+
+#: data/com.github.danrabbit.lookbook.desktop.in:7
+msgid "icons;iconography;"
+msgstr "icons;ícones;iconography;iconografia;"
```

Comment:
What is the reason for using the English words in the translation as well?

Reply:
Its to make the search broader. But i can removed them if you wish

Reply:
It's not for me to decide, but I'd assume someone using a localized interface to search using their language's terms rather than English.

Reply:
Makes sense. Removed them